devel/apache-arrow/Portfile

```diff
@@ -208,56 +198,119 @@ if {[string match "py*" ${subport}]} {
-
-    configure.args-append \
-                    -DPython3_EXECUTABLE=${python.bin} \
-                    -Dre2_DIR=${prefix}/lib/cmake/grpc/modules
+                    -DCMAKE_MAKE_PROGRAM:STRING=[portbuild::build_getmaketype]
+    }
 
     # macOS wheel:
     # https://github.com/apache/arrow/blob/master/ci/scripts/python_wheel_macos_build.sh
+    configure.args-replace \
+                    -DARROW_PYTHON:BOOL=OFF \
+                    -DARROW_PYTHON:BOOL=ON
+    configure.args-replace \
+                    -DARROW_RPATH_ORIGIN:BOOL=OFF \
+                    -DARROW_RPATH_ORIGIN:BOOL=ON
+    configure.args-append \
+                    -DPython3_EXECUTABLE:FILEPATH=${python.bin} \
+                    -Dre2_DIR:PATH=${prefix}/lib/cmake/grpc/modules
+
+    pre-build {
+        reinplace -E -q "s|^(set\\(PLASMA_STORE_SERVER\[\[:space:]]+\")(${prefix}/bin/plasma-store-server\"\\))|\\1${configure_destdir}\\2|" \
+            ${configure_destdir}${prefix}/lib/cmake/arrow/PlasmaConfig.cmake
+    }
+
     build.env-append \
-                    "PATH=${llvm_bin}:$env(PATH)" \
-                    "PYARROW_CMAKE_OPTIONS=[concat {*}${configure.pre_args} {*}${configure.args}]" \
+                    CCACHE_DIR=${workpath}/.ccache \
+                    "PATH=${workpath}/bin:$env(PATH)" \
+                    "PKG_CONFIG_PATH=/usr/lib/pkgconfig:${build.dir}/install/lib/pkgconfig" \
+                    PYARROW_BOOST_USE_SHARED=1 \
+                    PYARROW_BUILD_TYPE=${cmake_build_type} \
+                    PYARROW_BUNDLE_ARROW_CPP=1 \
+                    PYARROW_CMAKE_OPTIONS=[exec echo {*}${configure.pre_args}] \
                     PYARROW_INSTALL_TESTS=1 \
+                    PYARROW_CMAKE_GENERATOR=${cmake.generator} \
                     PYARROW_WITH_DATASET=1 \
                     PYARROW_WITH_FLIGHT=1 \
                     PYARROW_WITH_HDFS=1 \
                     PYARROW_WITH_ORC=1 \
                     PYARROW_WITH_PARQUET=1 \
                     PYARROW_WITH_PLASMA=1 \
-                    PYARROW_WITH_S3=1 \
-                    PYARROW_WITH_TENSORFLOW=1 \
-                    CCACHE_DIR=${workpath}/.ccache
+                    PYARROW_WITH_S3=0 \
+                    PYARROW_WITH_TENSORFLOW=1
 
     build.dir       ${worksrcpath}/python
-    build.target    build_ext
-    build.post_args \
-                    --with-dataset \
-                    --with-flight \
-                    --with-parquet \
-                    --with-plasma \
-                    --with-tensorflow \
-                    --build-type=${arrow_build_type} \
-                    --bundle-arrow-cpp bdist_wheel
+    build.cmd       ${python.bin} setup.py --no-user-cfg
+    build.target    bdist_wheel
+    build.post_args
 
     destroot.env-append \
-                    "PATH=${python.prefix}/bin:${llvm_bin}:$env(PATH)" \
+                    "${python.prefix}/bin:${workpath}/bin:$env(PATH)" \
                     CCACHE_DIR=${workpath}/.ccache
```

Comment:
Surely the PATH= is correct here ?

Reply:
Correct, and unnecessary. The `cmake` options take care of this. Fixed by deletion.